src/OpenLoco/Windows/Construction/Common.cpp

```diff
@@ -738,7 +739,7 @@ namespace OpenLoco::Ui::Windows::Construction
         // 0x0049EFEF
         static void drawRoadTabs(Window* self, Gfx::Context* context)
         {
-            auto company = CompanyManager::get(_playerCompany);
+            auto company = CompanyManager::get(getGameState().playerCompanies[0]);
```

Comment:
we already have a `getGameState().playerCompanies[0]` with `CompanyManger::getControllingId()` could also use `getPlayerCompany` which is a helper function for this.